_config.yml

```diff
@@ -5,7 +5,8 @@ baseurl: ""
 shared_baseurl: ""
 syntax_highlight_theme: "tomorrow-night"
 cdnurl: "https://cloud.tinymce.com/5/tinymce.min.js"
-codepen_url: "https://cloud.tinymce.com/5/tinymce.min.js?apiKey=qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc"
+tinymce_codepen_url: "https://cloud.tinymce.com/5/tinymce.min.js?apiKey=qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc&tinydrive=dev"
+tinydrive_codepen_url: "https://cdn.tiny.cloud/1/qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc/tinydrive/dev/tinydrive.min.js"
```

Comment:
Should this be using `dev`? As that's the v4 branch atm. Theoretically that should be fine but given this is meant to be the v5 docs I'm wondering if it should use `5-dev` instead?

Note: I'm assuming we should also update this to `5-stable` once it's actually released.